lib/keio_updater.rb

```diff
@@ -0,0 +1,28 @@
+require "cluster"
+require "services"
+
+Services.mongo!
+
+class KeioUpdater
+  def run
+    query = {
+      "ht_items.0": {"$exists": 1},
+      "ht_items.collection_code": "KEIO",
+      "ht_items.billing_entity": "hathitrust",
+    }
+
+    Cluster.where(**query).no_timeout.each do |cluster|
+      cluster.ht_items.each do |ht_item|
+        if ht_item.collection_code == "KEIO" && ht_item.billing_entity == "hathitrust"
+          ht_item.billing_entity = "keio"
```

Comment:
`ht_item.set_billing_entity` would also work, but same diff.

Reply:
Nope, not without changing `set_billing_entity` to public, which I didn't feel was called for.